pxtsim/runtime.ts

```diff
@@ -156,7 +156,7 @@ namespace pxsim {
 
         export async function promiseTimeout<T>(ms: number, promise: T | Promise<T>, msg?: string): Promise<T> {
             let timeoutId: any;
-            let res: () => void;
+            let res: (_?: any) => void;
```

Comment:
```suggestion
            let res: (v?: T | PromiseLike<T>) => void;
```

to keep in sync with util.ts:promiseTimeout / not use any